README.md

```diff
@@ -266,7 +266,17 @@ componentDidMount() {
 }
 ```
 
+### Customization
 
+It is possible to customize the icon used in the notification on Android.
+By default you can add a drawable resource to your package with the file name `music_control_icon` and the notification will use your custom icon. If you
```

Comment:
As you already know, this is an option. Why add a new function for that? It would only add complexity and redundancy to this module.

Also, if I were you, I would accept a `require` resource instead of just using a string

Reply:
Thanks for the review. I guess I really didn't make a good case for the function part of this PR, so let me try to do that here. One of the features I was looking for was the ability to specify the notification icon I want to use on the fly. Say my app plays various types of media, and right now it plays an audio book. I would like it to show a book icon in the notification. Then the user switches to listening to a podcast, and I would like to show a microphone icon in the notification instead. With just the custom option, this would not be possible, but having the function allows me to basically point the notification to a different resource on the fly, and I one of the ways I use this feature. I believe that justifies it being a function, but if you still believe it's not necessary, I can remove it from the PR.

Regarding using a string, I went the drawable resource route, because Android resources basically provide you with a free multi resolution support. So if you use PNGs, Android drawable resources allow you to generate files of resolutions that match various device resolutions. I think having a `require` resource is a good idea, but I think it should be in a separate function because it provides very different functionality and adds more complexity while losing out on built in multi resolution support. Let me know what you think about this.

Reply:
Thanks for clarifying. Is there any reason to create a new function? Can't this be an option in `setNowPlaying` instead?

Reply:
ok, the latest update to the PR should take care of this. let me know if there are any other issues